From f1890690264e7e5ce7f5b0a32d73d910ce2cbd73 Mon Sep 17 00:00:00 2001
From: Profpatsch <mail@profpatsch.de>
Date: Sun, 21 Jul 2024 16:08:34 +0200
Subject: [PATCH] Fix imports for new mtl

mtl decided to un-export `Control.Monad` and `Data.Monoid`, which in
itself is fine, but of course it breaks everything.
---
 src/PureNix/Convert.hs | 1 +
 src/PureNix/Main.hs    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/PureNix/Convert.hs b/src/PureNix/Convert.hs
index 83ff2d7..33b767a 100644
--- a/src/PureNix/Convert.hs
+++ b/src/PureNix/Convert.hs
@@ -5,6 +5,7 @@
 
 module PureNix.Convert (convert, ModuleInfo (..)) where
 
+import Control.Monad
 import Data.Bitraversable
 import qualified Data.Map as M
 import Data.Set (Set)
diff --git a/src/PureNix/Main.hs b/src/PureNix/Main.hs
index d393d8b..0ec3272 100644
--- a/src/PureNix/Main.hs
+++ b/src/PureNix/Main.hs
@@ -17,6 +17,7 @@ import qualified System.Exit as Sys
 import System.FilePath ((</>))
 import qualified System.FilePath as FP
 import System.IO
+import Control.Monad
 
 defaultMain :: IO ()
 defaultMain = do
-- 
2.44.1

